code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -17,7 +17,7 @@
 	new /obj/item/radio/headset/heads/captain/alt(src)
 	new /obj/item/radio/headset/heads/captain(src)
 	new /obj/item/storage/belt/sabre(src)
-	new /obj/item/storage/box/gunset/pdh(src) // SKYRAT EDIT CHANGE - ORIGINAL: new /obj/item/gun/energy/e_gun(src)
+	new /obj/item/storage/box/gunset/m1911_captains(src)//KEPLER EDIT
```

Comment:
non-negotiable, no silly comments, it's bubber edit.
don't make it confusing like this